### PR TITLE
Initial implementation of the plug-in

### PR DESCRIPTION
**Pull request by llucax**
This is the initial implementation. We still need to add more tests and documentation, but this will be done in follow-up PRs.

### CODE REVIEW

**Comment by llucax** [comment]
Enabled auto-merge. My idea is to merge this sooner than later, as it is tested and working locally, so I can make an early release that can be used by the microgrid API client. I plan to work on the tests and docs after this initial release.

**Comment by Marenz** [comment]
Tests failing unfortunately

**Comment by llucax** [comment]
Updated. Hopefully this time will pass. I also remove a file that sneaked in accidentally in the mkdocs commit.

**Comment by llucax** [comment]
This is now ready for a final review. It was tested all the way up to the SDK (this repo -> microgrid API bindings -> microgrid API client -> SDK) and it seems to work as expected.

File: .cookiecutter-replay.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "github_org": "frequenz-floss",
+    "license": "MIT",
+    "author_name": "Frequenz Energy-as-a-Service GmbH",
+    "author_email": "floss@frequenz.com",
+    "python_package": "setuptools_betterproto",
+    "pypi_package_name": "setuptools-betterproto",
+    "github_repo_name": "setuptools-betterproto",
+    "default_codeowners": "(like @some-org/some-team; defaults to a team based on the repo type)",
+    "_template": "gh:frequenz-floss/frequenz-repo-config-python",
+    "_repo_dir": "/home/luca/.cookiecutters/frequenz-repo-config-python/cookiecutter",
```
Comment (daniel-zullo-frequenz) [inline .cookiecutter-replay.json:18]: this won´t work for other user's path so we should probably amend it. I guess it needs to be done in repo-config

Comment (llucax) [reply .cookiecutter-replay.json:18]: Thanks, will remove!

Comment (llucax) [reply .cookiecutter-replay.json:18]: * https://github.com/frequenz-floss/setuptools-betterproto/pull/11

**Pull request merged**